src/mscorlib/shared/System/IO/Error.cs

```diff
@@ -16,6 +16,11 @@ namespace System.IO
     [Pure]
     internal static class Error
     {
+        internal static Exception GetStreamIsClosed()
```

Comment:
This one is good. It is not used in CoreCLR right now, but it will be later once we get more files in sync.